src/vs/editor/browser/widget/codeEditorWidget.ts

```diff
@@ -1360,6 +1360,7 @@ export class CodeEditorWidget extends Disposable implements editorBrowser.ICodeE
 			if (decorationOption.hoverMessage) {
 				opts.hoverMessage = decorationOption.hoverMessage;
 			}
+
 			newModelDecorations.push({ range: decorationOption.range, options: opts });
```

Comment:
Nit: Please don't touch unrelated files.

Reply:
Thanks, I've removed this.